src/main/java/models/games/AnotherGame.java

```diff
@@ -0,0 +1,24 @@
+package models.games;
+
+import models.GameTypes;
+import models.content.AnotherContent;
+import models.games.abstracts.AbstractGame;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+public class AnotherGame extends AbstractGame {
```

Comment:
Why not more the same functionality into the abstract class

Reply:
- [x]  Override method action into AbstractGame